src/api/parser/src/utils/html/sanitize.js

```diff
@@ -0,0 +1,59 @@
+// Sanitize HTML and prevent XSS attacks
+// <img src=x onerror=alert(1)> becomes <img src="x">
+
+const sanitizeHtml = require('sanitize-html');
+
+module.exports = (dirty) => {
+  return sanitizeHtml(dirty, {
+    // https://github.com/apostrophecms/sanitize-html#what-are-the-default-options
+    allowedTags: [
+      'a',
+      'b',
+      'blockquote',
+      'br',
+      'caption',
+      'code',
+      'del',
+      'div',
+      'em',
+      'figure',
+      'h1',
+      'h2',
+      'h3',
+      'h4',
+      'h5',
+      'h6',
+      'hr',
+      'i',
+      'iframe',
+      'img',
+      'li',
+      'ol',
+      'p',
+      'pre',
+      'strike',
+      'strong',
+      'table',
+      'tbody',
+      'td',
+      'th',
+      'thread',
+      'tr',
+      'ul',
+    ],
+    allowedSchemesByTag: { img: ['http', 'https', 'data'] },
+    allowedAttributes: {
+      iframe: ['src'],
+      img: ['src'],
+      a: ['href'],
+    },
+    allowedIframeHostnames: [
+      'www.youtube.com',
```

Comment:
We have a PR in flight right now that alters this file, let's not forget to update this too or later (@HyperTHD).

Reply:
I'll remember